Package.swift

```diff
@@ -72,7 +78,6 @@ let package = Package(
                 .define("_GNU_SOURCE", to: "1"),  // Fix build error for Swift 5.9 onwards
             ],
             linkerSettings: [
-                .linkedLibrary("curl"),
                 .linkedLibrary("sasl2"),
```

Comment:
Should we remove this here as well then?

Reply:
Okay we should look into building libsassl with SwiftPM then to avoid the system dependency.